src/components/NotFound.tsx

```diff
@@ -3,13 +3,16 @@ import { useNavigate } from 'react-router-dom';
 
 function NotFound() {
   const navigate = useNavigate();
+  const handleButtonClick = () => {
+    navigate('/jobs', { replace: true });
```

Comment:
I think it is better to navigate to the last page visit through the comma instead of a specific page.